core/scripts/cli/voting/retrieveRewards.js

```diff
@@ -0,0 +1,52 @@
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+const style = require("../textStyle");
+const batchRetrieveRewards = require("./batchRetrieveRewards");
+const getResolvedVotesByRoundId = require("./getResolvedVotesByRoundId");
+const inquirer = require("inquirer");
+
+module.exports = async (web3, voting) => {
```

Comment:
nit: slightly prefer naming functions and exporting them at the bottom. I think it just gives the reader a little more in-built documentation about what's going on.

Reply:
I agree, I gave a name to every method